pkg/networkservice/chains/nsmgr/unix_test.go

```diff
@@ -160,3 +164,49 @@ func Test_MultiForwarderSendfd(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 1, counter.Closes())
 }
+
+func Test_TimeoutRecvfd(t *testing.T) {
```

Comment:
Could you explain how this test checks the problem?

Reply:
Client makes a request with an expired token thus triggering timeout (`timeoutserver`).

Reply:
The test is incorrect. Refresh was not disabled. 

Please add test for recvfd package. Also, I think we do not need to use sandbox here.